src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,25 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+
+  &__card {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $ceil-size;
+  }
+
+  @media (max-width: 899px) {
```

Comment:
strange condition, I would change it. There is a bug because of it:
![image](https://user-images.githubusercontent.com/52370890/133998761-f4df3831-123e-45a7-bbd3-f48f54aec1a4.png)